test.js

```diff
@@ -17,10 +17,9 @@ describe('Plugin', function() {
   });
 
   it('should compile and produce valid result', function(done) {
-    var content = '@color: #4D926F; #header {color: @color;}';
     var expected = '#header {\n  color: #4d926f;\n}\n';
 
-    plugin.compile({data: content, path: 'style.less'}, function(error, result) {
+    plugin.compile({path: 'test_files\\style.less'}, function(error, result) {
```

Comment:
Please name the directory `test-fixtures`, and rather than check the files into the repo, programmatically write them before running the tests and delete them afterward. It would be ok to use the `fs` `Sync` methods in this case.